specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-06-01/machineLearningServices.json

```diff
@@ -1811,6 +1811,34 @@
       ],
       "x-ms-discriminator-value": "DataLakeAnalytics"
     },
+    "OnPrem": {
```

Comment:
is this service tag online already? otherwise, this will be treated as breaking change.

Reply:
The PR is marked as Draft. @leizhaoPS What the PR is ready for review?